core/trie/proof_test.go

```diff
@@ -183,113 +184,270 @@ func build3KeyTrie(t *testing.T) *trie.Trie {
 	return tempTrie
 }
 
-func build4KeyTrie(t *testing.T) *trie.Trie {
```

Comment:
Some of these deleted functions will be bring back with #2252.
For this PR they are unused.